GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -506,6 +518,34 @@ private static bool CheckConditionWithRetry(Func<bool> predicate, int retries, i
             return result;
         }
 
+        private EventMetadata GetProcessInteractionData(ConcurrentDictionary<string, PlaceHolderCreateCounter> collectedData, ref bool logToFile)
+        {
+            EventMetadata metadata = new EventMetadata();
+
+            if (collectedData.Count > 0)
+            {
+                int count = 0;

```

Comment:
I don't think you need the count variable at all.  Whippee less code! :-)

Reply:
We append the count onto `ProcessName` to sort processes by the ones that are the most active to least active. Now that the EventMetadata is nested, maybe it's not needed?